manifests/l/LazapDevelopment/Lazap/0.6.5/LazapDevelopment.Lazap.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: LazapDevelopment.Lazap
+PackageVersion: 0.6.5
+PackageLocale: en-US
+Publisher: Lazap Development

```

```suggestion
Publisher: lazap
```
This should match the value written to control panel